android/app/src/main/res/values-de/strings.xml

```diff
@@ -779,7 +779,7 @@
     <string name="elevation_profile_time">Dauer:</string>
     <string name="isolines_toast_zooms_1_10">Karte vergrößern, um Höhelinien sichtbar zu machen</string>
     <string name="downloader_loading_ios">Herunterladen</string>
-    <string name="download_map_title">Herunterladen der Weltkarte</string>
+    <string name="download_map_title">Lade die weltweite Übersichtskarte herunter</string>
```

Comment:
For the moment I would change it to the formal speech, because we used it in the other sentences. 
```suggestion
    <string name="download_map_title">Laden Sie die weltweite Übersichtskarte herunter</string>
```

Reply:
:+1: I used informal language because #8465 is likely to be merged. But one formal and one informal was not very consistent.